src/footprints/rotary.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+    nets: ['A', 'B', 'C', 'S1', 'S2', 'MP1', 'MP2'],
+    params: {
+        class: 'S'
```

Comment:
The class letter should be representative of the type of component this is. I'm guessing "R" for rotary? Can't recall if it's already taken by something... Or maybe "E" for encoder (but then also rename the whole thing)? Please check the others for availability.

Reply:
Set as "R", as i'm not familiar with the encoder "family".